internal/storage/v2/clickhouse/tracestore/reader.go

```diff
@@ -113,10 +113,13 @@ func (r *Reader) GetOperations(
 		if err := rows.ScanStruct(&operation); err != nil {
 			return nil, fmt.Errorf("failed to scan row: %w", err)
 		}
-		operations = append(operations, tracestore.Operation{
-			Name:     operation.Name,
-			SpanKind: operation.SpanKind,
-		})
+		o := tracestore.Operation{
+			Name: operation.Name,
+		}
+		if operation.SpanKind != "unspecified" {
```

Comment:
what's the motivation for this special casing? Not _storing_ the "unspecified" string would make sense, as a space saving measure, and re-inflate empty string into "unspecified" on reads.